examples/offline_inference_neuron.py

```diff
@@ -12,7 +12,7 @@
 
 # Create an LLM.
 llm = LLM(
-    model="openlm-research/open_llama_3b",
+    model="TinyLlama/TinyLlama-1.1B-Chat-v1.0",
```

Comment:
This model is smaller and can run on the smallest neuron instance.

Reply:
nice!